### PR TITLE
Styled coordinates and added bottom credit bar with links

### PR DESCRIPTION
**Pull request by scottaames**

### CODE REVIEW

**Comment by kaitlyndlee** [comment]
Looks like there are some conflicts

File: app/src/.gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.cache/
```
Comment (kaitlyndlee) [inline app/src/.gitignore:1]: Did you mean to add this in src?

Comment (scottaames) [reply app/src/.gitignore:1]: Yeah - it seems like git is adding things we don't want, so I wanted to make sure it was getting everything in that folder too.  You can have multiple of them from what I understand.

**Comment by scottaames** [comment]
> Looks like there are some conflicts

Fixed.

**Review by kaitlyndlee — APPROVED** [review]
Looks good!

**Review by cgm78 — APPROVED** [review]

**Pull request merged**